### ISSUE TITLE
[BottomNavigationView] French role description string translation is misleading

### ISSUE THREAD

**Issue by PomepuyN** [material-components/material-components-android#2783]
**Description:** When talkback is activated and the device is set to French, the content description of a `BottomNavigationView` item is "Tabulation" which is not right. It should be "Onglet". "Tabulation" means "indentation" in French.

**Expected behavior:**

![Screenshot_20220627_082039](https://user-images.githubusercontent.com/371298/175872598-366b47c9-a8ba-473c-b0f3-2648d8ed4126.png)

**Source code:**  

This line should be updated: https://github.com/material-components/material-components-android/blob/431e291b01a4862146bba73381cda749bbfe4363/lib/java/com/google/android/material/resources/res/values-fr/strings.xml#L18

**Minimal sample app repro:** 

Any sample with a `BottomNavigationView` will reproduce the issue

**Android API version:** Any

**Material Library version:** 1.5.0

**Device:** any

PS: If needed, I can open a Pull Request for that.

**Comment by drchen** [comment]
I reported this to the localization team. They will verify with the translators and update the string if they identify this is a valid request. : )

I'll close the issue for now. Feel free to reopen it if the translation is not updated in the next release. : )
